clarity/src/vm/docs/mod.rs

```diff
@@ -568,11 +575,105 @@ const XOR_API: SimpleFunctionAPI = SimpleFunctionAPI {
 ",
 };
 
+const BITWISE_XOR_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: None,
+    snippet: "^ ${1:expr-1} ${2:expr-2}",
+    signature: "(^ i1 i2...)",
+    description: "Alias for the `xor` function.  Returns the result of bitwise exclusive or'ing a variable number of integer inputs.",
+    example: "(^ 1 2) ;; Returns 3
+(^ 120 280) ;; Returns 352
+(^ -128 64) ;; Returns -64
+(^ u24 u4) ;; Returns u28
+(^ 1 2 4 -1) ;; Returns -8
+",
+};
+
+const BITWISE_AND_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("& (bitwise and)"),
+    snippet: "& ${1:expr-1} ${2:expr-2}",
+    signature: "(& i1 i2...)",
+    description: "Returns the result of bitwise and'ing a variable number of integer inputs.",
+    example: "(& 24 16) ;; Returns 16
+(& 28 24 -1) ;; Returns 24
+(& u24 u16) ;; Returns u16
+(& -128 -64) ;; Returns -128
+(& 28 24 -1) ;; Returns 24
+"
+};
+
+const BITWISE_OR_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("| (bitwise or)"),
+    snippet: "| ${1:expr-1} ${2:expr-2}",
+    signature: "(| i1 i2...)",
+    description: "Returns the result of bitwise inclusive or'ing a variable number of integer inputs.",
+    example: "(| 4 8) ;; Returns 12
+(| 1 2 4) ;; Returns 7
+(| 64 -32 -16) ;; Returns -16
+(| u2 u4 u32) ;; Returns u38
+"
+};
+
+const BITWISE_NOT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("~ (bitwise not)"),
+    snippet: "~ ${1:expr-1}",
+    signature: "(~ i1)",
+    description: "Returns the result of bitwise not, effectively reversing the bits of `i1`.",
+    example: "(~ 3) ;; Returns -4
+(~ u128) ;; Returns u340282366920938463463374607431768211327
+(~ 128) ;; Returns -129
+(~ -128) ;; Returns 127
+"
+};
+
+const BITWISE_LEFT_SHIFT_API: SimpleFunctionAPI = SimpleFunctionAPI {
+    name: Some("<< (binary left shift)"),
+    snippet: "<< ${1:expr-1} ${2:expr-2}",
+    signature: "(<< i1 u2)",
+    description: "Moves all the bits in `i1` to the left by the number of places specified in `u2`. 
```

Comment:
Please use `i2` instead of `u2`.  `u2` is a Clarity literal.

"Shifts all the bits in `i1` to the left by the number of places specified in `i2`.  The rightmost `i2` places are filled with zeros."

This is _not_ the equivalent of multiplication in Clarity.  Multiplication can overflow, leading to a runtime error.  This does not.